package.v7-diff.json

```diff
@@ -0,0 +1,8 @@
+{
+    "peerDependencies": {
+        "webdriverio": "~6 || ~7"
+    },
+    "devDependencies": {
+        "webdriverio": "^7.0.0"
+    }
+}
```

Comment:
I think this is not required, we can directly edit the devDepencie using `jq`. We need not change peer dependency since the one installed would be v7.

Reply:
* I was thinking the same, though if additional stuff is added in v7 it will create clutter in `test.yml` thus added this file. 
but we can remove it as well.

Reply:
Additional thing like what? Can you share an example?

Reply:
post discussion, we decided to have a separate branch for `v2` ( supporting wdio v6, v7) and this PR will be `master` or version `3.x.x` which will support v8